modules/arm_plugin/cmake/features.cmake

```diff
@@ -24,9 +24,13 @@ endif()
 set(ARM_COMPUTE_TARGET_ARCH "${ARM_COMPUTE_TARGET_ARCH_DEFAULT}" CACHE STRING "Architecture for ARM ComputeLibrary")
 set_property(CACHE ARM_COMPUTE_TARGET_ARCH PROPERTY STRINGS ${ARM_COMPUTE_TARGET_ARCHS})
 
+set(ARM_COMPUTE_SCONS_JOBS "" CACHE STRING "Number of simultaneous jobs to build ARM ComputeLibrary with")
+set(ARM_COMPUTE_WERROR "" CACHE STRING "Enable/disable the -Werror compilation flag for ARM ComputeLibrary")
```

Comment:
1. Do we need this option? Can we always disable warnings as errors for ACL
2. IF we actually need this option, let's make it [BOOL](https://cmake.org/cmake/help/latest/command/set.html)

Reply:
It's reasonable to disable the option completely. Fixed